src/components/ui/Partner.tsx

```diff
@@ -1,3 +1,50 @@
-export function Partner() {
-	return <div>TODO</div>;
+import Link from "next/link";
+
+type PartnerProps = {
+	logo: string // URL a logóhoz
+	name: string // a partner neve
+	address: string // a partner címe
+	tel: string // telefonszám
+	email: string // email cím
+	website: string // weboldal
+};
+
+export default function Partner({ logo, address, email, name, tel, website }: PartnerProps) {
+	// protokol kitörlése a címből
+	const shortUrl = website.replace(/(^\w+:|^)\/\//, '');
```

Comment:
Ez miért is kell?

Reply:
https://google.com -> google.com
Jobban néz ki, konzisztensebb a Figmával.

Reply:
Szerintem hogy milyen link van a figmával az kb. mindegy, a protokollt lehagyni meg véleményem szerint a nullánál is kevesebb értelme van (tippre senki se nézi ott van-e, ha ráhúzza az egeret egy linkre).

Reply:
megjelenített szöveg szépítése törlésre került